core-blocks/button/editor.scss

```diff
@@ -1,9 +1,11 @@
 .editor-block-list__block[data-type="core/button"] {
+
```

Comment:
Weird... why does it want spaces here, @ntwb?

Reply:
A few reasons:

• Due to the number of permutations and combinations of _where_ and _where not_ new empty lines should occur, I've an issue to audit all of these at https://github.com/WordPress-Coding-Standards/stylelint-config-wordpress/issues/65

• The lack off SCSS used currently in WordPress leading to not having an official "WordPress SCSS Coding Standards"

-----

Solution/Workaround:

Update the config used here to override the appropriate rules here in this PR and then follow up by updating `stylelint-config-wordpress`, either or both the CSS and SCSS configs and releasing a new version of `stylelint-config-wordpress`.